json/core-config.json

```diff
@@ -1,5 +1,6 @@
 {
   "batchingIntervalInSeconds": 600,
```

Comment:
We also need to remove all references of `bitcoinFee` in the repo, such as in `bitcoin-config.json` & `bitcoin-config-test.json`.